helm/rook-operator/Chart.yaml

```diff
@@ -9,6 +9,13 @@ description: File, Block, and Object Storage Services for your Cloud-Native Envi
 engine: gotpl
 home: https://github.com/giantswarm/rook-operator-app
 icon: https://s.giantswarm.io/app-icons/2/svg/rook-dark.svg
+keywords:
+- operator
+- rook
+- ceph
+kubeVersion: ">=1.16.0"
+maintainers:
+  - name: glitchcrab
```

Comment:
General idea behind `maintainers` is to have the upstream maintainers listed there (only if we know them)